web/modules/webspark/webspark_blocks/config/install/image.style.block_image_16_9_lge.yml

```diff
@@ -5,6 +5,8 @@ dependencies:
     - crop.type.16_9
   module:
     - crop
+_core:
```

Comment:
I think this entire key `_core`  added by the CMS must be removed from the PR.
Right @mlsamuelson , @bebaps, @tbutterf ?

Reply:
@juanmitriatti I've gone ahead and removed the `_core` and hash lines from the configs.